internal/metrics/metrics.go

```diff
@@ -58,11 +64,17 @@ type Meta struct {
 const (
 	BuildInfoGauge = "contour_build_info"
 
-	HTTPProxyTotalGauge     = "contour_httpproxy_total"
-	HTTPProxyRootTotalGauge = "contour_httpproxy_root_total"
-	HTTPProxyInvalidGauge   = "contour_httpproxy_invalid_total"
-	HTTPProxyValidGauge     = "contour_httpproxy_valid_total"
-	HTTPProxyOrphanedGauge  = "contour_httpproxy_orphaned_total"
+	DeprecatedHTTPProxyTotalGauge     = "contour_httpproxy_total"
+	DeprecatedHTTPProxyRootTotalGauge = "contour_httpproxy_root_total"
+	DeprecatedHTTPProxyInvalidGauge   = "contour_httpproxy_invalid_total"
+	DeprecatedHTTPProxyValidGauge     = "contour_httpproxy_valid_total"
+	DeprecatedHTTPProxyOrphanedGauge  = "contour_httpproxy_orphaned_total"
+
+	HTTPProxyTotalGauge     = "contour_httpproxy"
```

Comment:
Could you add some tests to cover that the deprecated and new metric names are the same value? We should add this coverage to ensure we don't break anyone who is already using these metrics before we remove them

Reply:
yes. updated an existing test.